python/README.md

```diff
@@ -0,0 +1,87 @@
+# README
```

Comment:
Awesome, but I did not see a similar file for the Matlab wrapper. Anything you need to say there?

Reply:
Nothing changed for MATLAB wrapper, so I don't know what should I put there...